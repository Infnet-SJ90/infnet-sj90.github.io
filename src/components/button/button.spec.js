import React from 'react';
import renderer from 'react-test-renderer';
import Button from './index';
import { mount } from 'enzyme';

const rendered = mount(<Button>Button Component</Button>);
const snapshot = renderer.create(<Button>Button Component</Button>).toJSON();

it('Should Button tag return output correctly', () => {
  expect(rendered.text()).toBe('Button Component');
});

it('Should Button match to snapshot', () => {
  expect(snapshot).toMatchSnapshot();
});
