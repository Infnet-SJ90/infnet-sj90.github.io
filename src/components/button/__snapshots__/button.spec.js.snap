// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Button match to snapshot 1`] = `
<div>
  <button
    className="btn btn-secondary"
    onClick={[Function]}
  >
    Button Component
  </button>
</div>
`;
